### PR TITLE
Catch requests error while send subtitle notifications

### PR DESCRIPTION
**Pull request by fernandog**
@duramato @ratoaq2

Fixes https://github.com/pymedusa/Medusa/issues/1980

### CODE REVIEW

File: medusa/tv/episode.py [hunk 1]
```diff
@@ -269,7 +271,10 @@ def download_subtitles(self, lang=None):
                         ep=(episode_num(self.season, self.episode) or
                             episode_num(self.season, self.episode, numbering='absolute')))
 
-            notifiers.notify_subtitle_download(self.pretty_name(), subtitle_list)
+            try:
+                notifiers.notify_subtitle_download(self.pretty_name(), subtitle_list)
```
Comment (p0psicles) [inline medusa/tv/episode.py]: Honestly I don't know how a notification can ever throw an exception. That smells like bad design.
As a notification is only needed, when you have something a notify on. Maybe using some kind of callback, can fix this.

Comment (p0psicles) [inline medusa/tv/episode.py]: Never mind. I thought it was pNotification.

**Comment by medariox** [comment]
Was this tested at least once? RequestException doesn't catch socket.gaierror.

**Comment by labrys** [comment]
@fernandog when you're not sure you can easily test 
```python
>>> try:
	raise socket.gaierror
except requests.exceptions.RequestException:
	pass

Traceback (most recent call last):
  File "<pyshell#19>", line 2, in <module>
    raise socket.gaierror
socket.gaierror
```

**Pull request merged**